crates/bevy_app/src/task_pool_options.rs

```diff
@@ -0,0 +1,157 @@
+use bevy_ecs::Resources;
+use bevy_tasks::{AsyncComputeTaskPool, ComputeTaskPool, IOTaskPool, TaskPoolBuilder};
+
+fn clamp_usize(value: usize, min: usize, max: usize) -> usize {
```

Comment:
Clamp is a big enough missing piece in the rust std that it probably makes sense to expose it in `bevy_math`. Can we add a generic version there and then consume it here?

Reply:
Ok, I'll copy/paste num-traits implementation (licensed MIT) with a note as to where it came from

Reply:
https://doc.rust-lang.org/std/cmp/trait.Ord.html#method.clamp

it's available on nightly, is it possible to switch to the default impl for nightly users?

Reply:
Ooh I didn't know that. Definitely good to know. In general I would prefer to wait for nightly features to land in stable. Otherwise we'll need to maintain multiple code paths, which makes it harder to debug and repro potential issues.